src/core/email/templates/progress-report-status-changed.template.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Button,
+  Column,
+  HideInText,
+  Section,
+  Text,
+} from '@seedcompany/nestjs-email/templates';
+import { startCase } from 'lodash';
+import { DateTime } from 'luxon';
+import { EmailReportStatusNotification } from '../../../components/progress-report/workflow/progress-report-workflow.service';
+import { fullName } from '../../../components/user/fullName';
+import { EmailTemplate, Heading } from './base';
+import { useFrontendUrl } from './frontend-url';
+
+export function ProgressReportStatusChanged({
+  changedBy,
+  project,
+  languageEngagement,
+  report,
+  reportQtrLabel,
+  newStatusVal,
+  previousStatusVal,
+}: EmailReportStatusNotification) {
+  const projectUrl = project?.id
+    ? useFrontendUrl(`/projects/${project.id}`)
+    : useFrontendUrl(`/projects/no-project-found`);
```

Comment:
I don't think there should ever be a case where we don't have a project. I would enforce that a project is required for this template. Ensuring that upstream caller has good data is better than forwarding a dead link to users.